windows/win_unzip.ps1

```diff
@@ -57,17 +57,36 @@ $recurse = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "recurse" -defaul
 $rm = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "rm" -default "false")
 
 If ($ext -eq ".zip" -And $recurse -eq $false) {
-    Try {
-        $shell = New-Object -ComObject Shell.Application
-        $zipPkg = $shell.NameSpace([IO.Path]::GetFullPath($src))
-        $destPath = $shell.NameSpace([IO.Path]::GetFullPath($dest))
-        # 20 means do not display any dialog (4) and overwrite any file (16)
-        $destPath.CopyHere($zipPkg.Items(), 20)
-        $result.changed = $true
+    If (-Not (Get-WindowsFeature -Name 'Server-Gui-Shell')) {
```

Comment:
Thanks @schwartzmx for your review.

Yes I agree installing new feature should not be part of win_unzip module.
On any version of Windows 2012(including server core) - dotnet framework 45 will be preinstalled by OS.
Ref: https://msdn.microsoft.com/en-us/library/8z6watww(v=vs.110).aspx 
Also, I just verified that the code to check and install dot net feature can be safely removed.

I shall submit a new patch with the updated changes. 
Kindly do let me know if any further changes required.